OtripleS.Web.Api.Tests.Unit/Services/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.Add.cs

```diff
@@ -0,0 +1,126 @@
+﻿//---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+//----------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.CalendarEntryAttachments;
+using OtripleS.Web.Api.Models.CalendarEntryAttachments.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.CalendarEntryAttachments
+{
+    public partial class CalendarEntryAttachmentServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnAddWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            CalendarEntryAttachment randomCalendarEntryAttachment = CreateRandomCalendarEntryAttachment();
```

Comment:
You don't need this line completely when you're creating `someVariable` so you should say:

```csharp
 CalendarEntryAttachment someCalendarEntryAttachment= CreateRandomCalendarEntryAttachment();
```

Reply:
Do the same for the rest.

Reply:
Resolved in all exceptions for Add, Remove and RetrieveById